include/rdma/fabric.h

```diff
@@ -297,6 +297,7 @@ struct fi_info {
 	uint64_t		caps;
 	uint64_t		mode;
 	uint32_t		addr_format;
+	uint32_t		version;
```

Comment:
I'm going to send an e-mail to the mailing list so we can discuss this during tomorrows telecon. I was trying to find a way to use the version parameter to `fi_getinfo`, but didn't see a clear one. I can work on trying to figure a better way out after we decide which method is worth investigating. For now I just wanted to get something out there so people can look at it for discussion tomorrow.